.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Please make sure to **read the following list** before creating a new issue:
+
+* This issue tracker is not supposed to be used for questions on how to use visjs. It is intended to be used for bug reports and feature requests! In case you face yourself with a usage question, then post your issue e.g. on [stackoverflow](https://stackoverflow.com/questions/tagged/vis.js) tagged with "vis.js".
```

Comment:
I think we should not be too strict here! We get a lot of interesting question, we only want to get rid of the stupid ones. (Maybe we should write: "No stupid questions please", lol)